matrix/copy_transform.py

```diff
@@ -0,0 +1,62 @@
+def rotate_clockwise(matrix):
+    new = []
+    for row in reversed(matrix):
+        for i, elem in enumerate(row):
+            try:
+                new[i].append(elem)
+            except IndexError:
+                new.insert(i, [])
+                new[i].append(elem)
+    return new
+
+def rotate_counterclockwise(matrix):
+    new = []
+    for row in matrix:
+        for i, elem in enumerate(reversed(row)):
+            try:
```

Comment:
Oh i didn't know it was a list.
Ok but can we think of a way without using try and catch?
I'll accept the PR, but I am looking for a way to simplify this.